tests/storage/hotplug.go

```diff
@@ -383,6 +383,40 @@ var _ = SIGDescribe("Hotplug", func() {
 		}, 90*time.Second, 2*time.Second).Should(Succeed())
 	}
 
+	waitForAttachmentPodToRun := func(vmi *kubevirtv1.VirtualMachineInstance) {
+		namespace := vmi.GetNamespace()
+		uid := vmi.GetUID()
+
+		labelSelector := fmt.Sprintf(v1.CreatedByLabel + "=" + string(uid))
+
+		pods, err := virtClient.CoreV1().Pods(namespace).List(context.Background(), metav1.ListOptions{LabelSelector: labelSelector})
+		Expect(err).ToNot(HaveOccurred(), "Should list pods")
+
+		var virtlauncher *corev1.Pod
+		for _, pod := range pods.Items {
+			if pod.ObjectMeta.DeletionTimestamp == nil {
+				virtlauncher = &pod
+				break
+			}
+		}
+		Expect(virtlauncher).ToNot(BeNil(), "Should find running virtlauncher pod")
+		Eventually(func() bool {
+			podList, err := virtClient.CoreV1().Pods(vmi.Namespace).List(context.Background(), metav1.ListOptions{})
```

Comment:
@awels I see that you've previously invoked the VMI namspace using `namespace := vmi.GetNamespace()`, why did you decide to directly read the namespace here using `vmi.Namespace`?

Reply:
I probably copied the podList from somewhere else, but good point, I can just use the namespace I read already